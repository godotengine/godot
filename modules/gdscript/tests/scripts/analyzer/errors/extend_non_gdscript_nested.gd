class Foo extends RefCounted.Bar:
	pass

func test():
	print('not ok')
