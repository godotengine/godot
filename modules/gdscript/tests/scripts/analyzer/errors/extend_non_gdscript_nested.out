GDTEST_ANALYZER_ERROR
>> ERROR at line 1: Cannot get nested types for extension from non-GDScript type "RefCounted".
